.travis.yml

```diff
@@ -42,7 +42,7 @@ env:
     - ROS_DISTRO=jade   PRERELEASE=true PRERELEASE_REPONAME=i_do_not_exist  EXPECT_EXIT_CODE=1 PRERELEASE_DOWNSTREAM_DEPTH=1  # Intended to fail, to test the capability of capturing Prerelease Test failure.
     - ROS_DISTRO=kinetic AFTER_SCRIPT='catkin --version'
     - ROS_DISTRO=kinetic NOT_TEST_BUILD='true' DEBUG_BASH='true' VERBOSE_OUTPUT='false'
-    - ROS_DISTRO=lunar ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - ROS_DISTRO=lunar ROS_REPO=ros-shadow-fixed
```

Comment:
Is there any issue in even shortening the value as "shadow-fixed" (without `ros-`)?